artifactory/utils/argsutils.go

```diff
@@ -0,0 +1,91 @@
+package utils
+
+import (
+	"errors"
+	"fmt"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+	"strings"
+)
+
+// Removes the provided flag and value from the command arguments
+func RemoveFlagFromCommand(args *[]string, flagIndex, flagValueIndex int) {
+	argsCopy := *args
+	// Remove flag from Command if required.
+	if flagIndex != -1 {
+		argsCopy = append(argsCopy[:flagIndex], argsCopy[flagValueIndex+1:]...)
+	}
+	*args = argsCopy
+}
+
+// Find value of required CLI flag in Command.
+// If flag does not exist, returned indexes are -1 and error is nil.
```

Comment:
...returned indexes are -1  and error is nil. --> ...the returned index is -1 and nil is returned as the error.

Reply:
Done